README.md

```diff
@@ -3,8 +3,13 @@
 
 ## URL
 
-This is currently setup as [aforward.github.io](https://aforward.github.io)
-but will move to `eclipse-pass.github.io` within [PASS ORG](https://github.com/eclipse-pass).
+This repo is available at the following URLs.
+
+* [eclipse-pass.github.io](https://eclipse-pass.github.io) - Underlying GitHub URL
```

Comment:
I seem to remember that once GitHub Pages is configured to use a custom URL the site is no longer available at the original github.io domain. Why don't we just wait until the domains have settled before merging this.

Reply:
The old URL still works, but it will _also_ be configured to redirect to the custom domain.  I like keeping the underlying URL there for troubleshooting purposes for those not familiar with GitHub Pages.

For example,

https://aforward.github.io redirect to https://eclipse-pass.anunknown.dev

(the example I set up before reaching out to EF).